ui/transactions.go

```diff
@@ -1,8 +1,10 @@
 package ui
 
 import (
+	"image"
```

Comment:
Similar to the transactions details page, this should have a clear file name. It could be "transactions_page.go"